client/reader/start/main.jsx

```diff
@@ -1,20 +1,43 @@
 // External dependencies
```

Comment:
We need to change the guidelines then because the suggestion of `reader/reader-start` is silly. The alternative, based on looking at `my-sites`, is to redo the classes to just use `start`, which seems dangerous. 

Looking through my-sites, it's using base classes with pretty generic sounding names, with no scoping against my-sites. `post-image` for instance. At least `post` is scoped inside `post-list`, but i really feel like these should all be inside a `my-sites` class scope for safety. If these components were ever promoted to `components` because they're actually generic and designed to be generic, we could drop the section from the selector.

I understand this doesn't fit the guideline, but the guideline is dangerous and needs to be revised.